packages/release/release-tmpfiles.conf

```diff
@@ -15,6 +15,5 @@ Z /var/lib/systemd 0755 root root -
 z /var/lib/systemd/random-seed 600 root root -
 R /var/lib/systemd/linger
 D /var/lib/systemd/linger 0700 root root -
-d /etc/sysctl.d 0700 root root -
```

Comment:
I'm inclined to keep this for now, and just have any units that need this directory early arrange to create it for themselves.

Right now `corndog` also needs this directory, and doesn't create it, and it's conceptually weird for `corndog` to depend on `netdog` (or related units) for this.

It's outside the scope of this change, but I'd like to explore a better mechanism, like `etc.target`, where we can collect all the units needed to populate `/etc`, and we order almost everything after that, to stop the madness of every early unit needing to depend on `selinux-policy-files.service` or run the risk subtle breakage.

`prepare-sysctl.service` seems like a similar dependency where more units may end up needing it over time.